### PR TITLE
PyMC tutorial - an initial version for review

### PR DESCRIPTION
**Pull request by daslu**
Hello @behrica.

Here is my first attempt to contribute to `clojure-data-tutorials` - a tutorial for [PyMC](https://www.pymc.io/) probabilistic programming.

The devcontainers approach seems to be very helpful in this case, as a few of us were having trouble sharing Clojure+PyMC code across different platforms.

The text is still a draft.

I'd appreciate any comments.

### CODE REVIEW

**Comment by daslu** [comment]
The Github checks report a strange failure:
```bash
error: Your local changes to the following files would be overwritten by checkout:
	docs/index.html
	docs/notebooks/toc.edn
```

**Comment by behrica** [comment]
> The Github checks report a strange failure:
> 
> ```shell
> error: Your local changes to the following files would be overwritten by checkout:
> 	docs/index.html
> 	docs/notebooks/toc.edn
> ```

I think this need to be fixed in a similar way then we did for Noj.
The build should not try to commit the changes do "docs"

**Comment by behrica** [comment]
I fixed teh build on "main".
So if you re-base or merge you will get the change.

then the build should pass.

Now "all tutorials" are again re-rendered on any commit.

This is lsow, but "correct" in my view.
The "docs" folder is not anymore in GIT.

**Comment by daslu** [comment]
Thanks, I'll try now.

**Comment by daslu** [comment]
Thanks for the quick help, @behrica.
The tests are passing.

I just added a slight change to the comments.

Would you like to review my draft, at least regarding the format and workflow?

File: projects/stats/pymc/.devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ENTRYPOINT ["bash"]
+
+
+# Prepare clojure tools
+RUN clojure -Ttools list && \
+clojure -Ttools install io.github.seancorfield/clj-new '{:git/tag "v1.2.404" :git/sha "d4a6508"}' :as clj-new && \
+clojure -Ttools install-latest :lib io.github.seancorfield/deps-new :as new && \
+clojure -Ttools list
+
+RUN sudo apt-get update && \
```
Comment (behrica) [inline projects/stats/pymc/.devcontainer/Dockerfile:31]: One way to setup the python. There are several options, but this is OK.

File: projects/stats/pymc/.devcontainer/devcontainer.json
Comment (behrica) [inline projects/stats/pymc/.devcontainer/devcontainer.json:1]: looks good to me

**Comment by behrica** [comment]
looks good to me.
Do you want too add more content ?

**Comment by daslu** [comment]
Thanks.

It is a draft. I will try to work with another person on making it a more full tutorial and maybe rearrange it as a book with a few small chapters.

If that is ok with you, it will be great to merge it so that the draft can be shared online.

File: projects/stats/pymc/.devcontainer/Dockerfile (hunk 1 above)
Comment (daslu) [reply projects/stats/pymc/.devcontainer/Dockerfile:31]: Thanks

**Comment by daslu** [comment]
@behrica Thanks for merging this.

Looks like the merge did not invoke any other actions, and the website does not include the new tutorial:
image

**Comment by daslu** [comment]
Thanks for merging, @behrica.

I deleted my previous comment, which was a misunderstanding I had.

**Comment by daslu** [comment]
I still need to figure out why the link is broken:
https://scicloj.github.io/clojure-data-tutorials/projects/stats/pymc/index.html

**Comment by daslu** [comment]
Oh, I just realized I should have updated `.github/workflows/main.yml` with a command to render this.

**Comment by behrica** [comment]
Sorry, indeed. 
I forgot to tell you.
Not sure, if we can fully automate this. I will have a look

**Comment by behrica** [comment]
@daslu 

I simplified the "build".
see here:
https://github.com/scicloj/clojure-data-tutorials/commit/297d7dc13379773ffb8153c97dff04c360c470c1

**Comment by daslu** [comment]
The new build setup looks great, thanks.

**Pull request merged**